### DISCUSSION TITLE
How to switch/migrate from filesystem to s3 without loosing any job or history

### DISCUSSION THREAD

**Discussion by tushar9953** [jhuckaby/Cronicle#563] [General]
Currently my cronicle is using filesystem to store its data. Now, wish to migrate it to s3. What is the best possible practice to migrate everything without losing any job data or user data?

Thanks in advance :)

**Comment by jhuckaby** [comment]
https://github.com/jhuckaby/Cronicle/blob/master/docs/CommandLine.md#storage-migration-tool
